### PR TITLE
rest of text changes to virtual card settings page

### PR DESCRIPTION
**Pull request by alanna**

### CODE REVIEW

File: components/edit-collective/sections/virtual-cards/VirtualCardFilters.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const VirtualCardFilters = ({
       {isCollectiveFilter && (
         <FilterContainer mr={[0, '8px']} mb={['8px', 0]}>
           <FilterLabel htmlFor="virtual-card-filter-collective">
-            <FormattedMessage id="VirtualCard.Collective" defaultMessage="Collective" />
+            <FormattedMessage id="VirtualCard.Collective" defaultMessage="Filter" />
```
Comment (znarf) [inline components/edit-collective/sections/virtual-cards/VirtualCardFilters.js:57]: Doesn't seem a good label. Let's roll back that one.

File: components/edit-collective/sections/HostVirtualCards.js [hunk 2]
```diff
@@ -221,15 +220,8 @@ const HostVirtualCards = props => {
       </SettingsTitle>
       <Box>
         <SettingsSectionTitle>
-          <FormattedMessage id="Host.VirtualCards.Settings.Title" defaultMessage="Settings and Policy" />
+          <FormattedMessage id="Host.VirtualCards.Settings.Title" defaultMessage="Settings" />
```
Comment (znarf) [inline components/edit-collective/sections/HostVirtualCards.js:223]: Why not but I don't see why it's better to have this section named "Settings" when it contains "Settings and Policy".

Comment (SudharakaP) [reply components/edit-collective/sections/HostVirtualCards.js]: @znarf : Yes, I've reverted this back as well. 😄

**Comment by SudharakaP** [comment]
@znarf : Corrected the issues you mentioned. Is this good to merge? 😄

**Comment by alanna** [comment]
@znarf is this good to go now?

**Review by znarf — APPROVED** [review]
Good for me.

**Pull request merged**